README.md

```diff
@@ -1,53 +1,41 @@
 # Project Name
 
-(short, 1-3 sentenced, description of the project)
+ACS Calling Sample for WPF.
 
 ## Features
 
 This project framework provides the following features:
 
-* Feature 1
-* Feature 2
-* ...
+* Embeds the ACS Calling Tutorial web application inside of a desktop application.
 
 ## Getting Started
 
 ### Prerequisites
 
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
+- [npm](https://www.npmjs.com/get-npm)
+- [Node.js](https://nodejs.org/en/download/)
+- [Microsoft Edge (Chromium) Canary Chanel](https://www.microsoftedgeinsider.com/en-us/download)
+- [Visual Studio 2017 or later](https://visualstudio.microsoft.com/)
+- .NET Framework. It can be installed from Visual Studio Installer.
```

Comment:
It would be better inform that the VS workload required: Microsoft.VisualStudio.Workload.ManagedDesktop

https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-enterprise?view=vs-2019&preserve-view=true#net-desktop-development